lib/all-badge-examples.js

```diff
@@ -310,6 +310,46 @@ const allBadgeExamples = [
         title: 'continuousphp',
         previewUri: '/continuousphp/git-hub/doctrine/dbal/master.svg'
       },
+      {
+        title: 'Discourse topics',
+        previewUri: '/discourse/meta.discourse.org/topics.svg',
```

Comment:
Do Discourse servers always run `https`?

Reply:
This is a good question. Let me find out.

Reply:
I will switch this to http. The library itself should handle the redirect if needed.

Reply:
If https is always supported, defaulting to that would be better. If https is sometimes unavailable, the way we'd handle that is to include the scheme in the badge URL.

Ideal would be: always use https, with no option for http, just like you have. Let's stick with that if it's always supported.

Reply:
https is not guaranteed, but is typically the case. I will see how it's done currently by using the badge URL to determine the scheme.